packages/visx-demo/src/sandboxes/visx-xychart/ExampleControls.tsx

```diff
@@ -720,6 +792,11 @@ export default function ExampleControls({ children }: ControlsProps) {
         input[type='radio'] {
           height: 10px;
         }
+        .pattern-lines {
```

Comment:
this currently pattern-only svg has a non-zero height on `/xychart`, this fixes it.